Demo/Demo/SwiftUIComponents/PayPalWebViews/PayPalWebOrderCompletionView.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+
+struct PayPalWebOrderCompletionView: View {
```

Comment:
This class looks pretty much the exact same as [CardPaymentOrderCompletionView.swift](https://github.com/paypal/paypal-ios/blob/main/Demo/Demo/SwiftUIComponents/CardPaymentViews/CardPaymentOrderCompletionView.swift). The views should be re-usable across payment method/ view model types.

This way if we want to update this view, we do it in 1 place versus multiple.

Reply:
See comment below.